docs/installing.adoc

```diff
@@ -285,7 +309,7 @@ export BOX=<box-name>
 ----
 ====
 
-[[set-up-adb-behind-an-http-proxy]]
+[[setup-http-proxy]]
 == Step 6. (Optional) Set up ADB behind an HTTP proxy
```

Comment:
`setup-http-proxy` looks incorrect. The section represents `how to setup ADB behind a proxy setup`